### PR TITLE
[SYCLomatic] Add dnn help function test case and migration test case

### PR DESCRIPTION
**Pull request by intwanghao**
Including: binary, reduction, batch_normalization, normalization, convolution

### CODE REVIEW

**Review by ziranzha — APPROVED** [review]
LGTM

**Review by ShengchenJ — APPROVED** [review]
LGTM

File: help_function/src/dnnl_utils_activation.cpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ void test1() {
     auto s = (handle.activation_forward(desc, alpha, dataTensor, data, beta,
                                         outTensor, out),
               0);
-
+    dev_ct1.queues_wait_and_throw();
```
Comment (zhimingwang36) [inline help_function/src/dnnl_utils_activation.cpp:92]: Please create a JIRA to followup the test update,  wait() here is not neccssary if oneDNN/compiler fix the syn issue for host_task in in-order sycl::queue in CPU device.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**